plugins/serverless/src/tasks/provision.ts

```diff
@@ -0,0 +1,60 @@
+import { ToolKitError } from '@dotcom-tool-kit/error'
+import { hookFork, styles } from '@dotcom-tool-kit/logger'
+import { Task } from '@dotcom-tool-kit/types'
+import { ServerlessSchema } from '@dotcom-tool-kit/types/lib/schema/serverless'
+import { VaultEnvVars } from '@dotcom-tool-kit/vault'
+import { spawn } from 'child_process'
+
+export default class ServerlessProvision extends Task<typeof ServerlessSchema> {
+  static description = 'Provisions a job on AWS'
+
+  static defaultOptions = {
```

Comment:
**issue:** just realised the [readme](https://github.com/Financial-Times/dotcom-tool-kit/tree/main/plugins/serverless) needs to be updated with the new task and options